contracts/zero-ex/foundry.toml

```diff
@@ -1,7 +1,34 @@
 [profile.default]
 src = 'contracts/src'
 out = 'foundry-artifacts'
-test = 'tests'
+test = 'contracts/test/foundry/local'
 libs = ["contracts/deps/", "../utils/contracts/src/"]
-remappings = ['@0x/contracts-utils/=../utils/', '@0x/contracts-erc20/=../erc20/', 'src/=./contracts/src']
+remappings = [
+    'forge-std/=contracts/deps/forge-std/src/',
+    'ds-test/=contracts/deps/forge-std/lib/ds-test/src/',
+    '@0x/contracts-utils/=../utils/',
+    '@0x/contracts-erc20/=../erc20/',
+    'src/=./contracts/src',
+    '@0x/contract-addresses/=../../packages/contract-addresses',
+    'samplers/=contracts/test/samplers',
+]
+cache_path = 'foundry-cache'
```

Comment:
Let's not configure custom cache path as this breaks slither (I know we haven't plugged it in yet but we will soon) plus I see nothing else using `cache` folder which is the foundry default.